src/bricks/transformers/ephemeralForm/EphemeralForm.test.tsx

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright (C) 2023 PixieBrix, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { getFormDefinition } from "@/contentScript/messenger/api";
+import { render, screen } from "@testing-library/react";
+import React from "react";
+import EphemeralForm from "./EphemeralForm";
+import { waitForEffect } from "@/testUtils/testHelpers";
+
+jest.mock("@/contentScript/messenger/api", () => ({
+  getFormDefinition: jest.fn(),
+}));
+
+const getFormDefinitionMock = jest.mocked(getFormDefinition);
+
+describe("EphemeralForm", () => {
+  it("shows field titles", async () => {
+    getFormDefinitionMock.mockResolvedValue({
+      schema: {
+        title: "Test Form",
+        type: "object",
+        properties: {
+          foo: {
+            type: "string",
+          },
+        },
+      },
+      uiSchema: {},
+      cancelable: false,
+      submitCaption: "Submit",
+      location: "modal",
+    });
+
+    const { asFragment } = render(<EphemeralForm />);
+
+    await waitForEffect();
+
+    // https://github.com/pixiebrix/pixiebrix-extension/pull/4913#issuecomment-1400379452
+    expect(screen.getByLabelText("foo")).not.toBeNull();
```

Comment:
Since this form is generated by the library, this is probably the best we can do here:
```suggestion
    expect(screen.getByText("foo")).toBeVisible();
    expect(screen.getByRole("textbox")).toBeVisible();
```

Reply:
The "custom" label in `RjsfSelectWidget` is only adding a `*` for required fields, and the label in `ImageCropWidget` is doing nothing:
<img width="447" alt="image" src="https://github.com/pixiebrix/pixiebrix-extension/assets/2801308/c245c06a-0416-40a6-90fd-2885bffe1879">

I think we could also remove these, and add the field label explicitly in our `FieldTemplate`, with any `*` logic for required fields, and explicitly add the `htmlFor` prop to the label to link it properly to the field.

Reply:
Good callout, I forgot there was a possibility we were overriding the select widget and that's why it wasn't getting a `for` prop. 

In that case, the test should ideally be `expect(screen.getByRole("textbox", { name: "foo" })).toBeVisible()`

Reply:
I think that if htmlFor isn't working, it should (eventually) be fixed. I can apply your suggested assertion here, but if `label` isn't connected to a field that's a bug

Reply:
> I think that if htmlFor isn't working, it should (eventually) be fixed. I can apply your suggested assertion here, but if `label` isn't connected to a field that's a bug

Agreed. And it looks like it's our bug, not RJSF. I suggest following @BLoe's recommendation. Updating `FieldTemplate` to include the `FormLabel`, passing the appropriate `htmlFor` to the label, and removing the `FormLabel` from the custom widgets should resolve the issue and allow us to make the correct assertion.

Reply:
It's not clear to me exactly what the expected behavior should be, with how we're using `JsonSchemaForm`. We're overriding several fields and widgets and also passing in a `FieldTemplate` that doesn't include the label. I don't even really understand how the label is properly being rendered in the default text widget, I wouldn't have expected the default widgets to include the label, I would have thought we need to include it in our `FieldTemplate` for it to show up at all.

It seems sketchy to me to depend on this weird "implicit" behavior where we expect the default text widget to also render it's own label and also attach it properly using `htmlFor`. Why not just render the labels explicitly in `FieldTemplate` so that we have full control, and make sure the `for` prop is working correctly?

Reply:
Ok so I briefly tried to make these changes, and it turns out there are two labels rendered now and everything seems messed up... 🤦 

This will take more dedicated time to understand better how the Rjsf FieldTemplate concept works with the default Rjsf widgets and field labels.

Reply:
I can take a look at this after my current issue.

Reply:
Took some digging, but I figured out that the Widgets control displaying labels, not the FIeldTemplate. 

I could theoretically used `React.Children.map` and `React.cloneElement` to set the `displayLabel` to `false` in all children of the custom FieldTemplate so that only the FieldTemplate label would render, but that's a huge hack that I don't think we want. I also believe it won't be a problem after the RJSF 5 upgrade is complete.